kogito-kafka-quickstart-springboot/README.md

```diff
@@ -16,15 +16,66 @@ This example shows
 	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
 
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties2.png"></p>
+
+* Diagram Properties
```

Comment:
What do you want to show in this diagram ?

Reply:
Fixed @radtriste